File: 1176.c
#include <stdio.h>

int main() {
    int i, n, cases;
    long long int fib[61];
    
    fib[0] = 0;
    fib[1] = 1;
    for (i = 2; i <= 60; i++) {
        fib[i] = fib[i-1] + fib[i-2];
    }

    scanf("%d", &cases);

    for (int j = 0; j < cases; j++) {
        scanf("%d", &n);
        printf("Fib(%d) = %lld\n", n, fib[n]);
    }

    return 0;
}

File: 1176.py
fib = [0, 1]  # Fib(0) = 0, Fib(1) = 1

for i in range(2, 61):
    fib.append(fib[i - 1] + fib[i - 2])  # Adiciona o próximo número de Fibonacci

cases = int(input())

for _ in range(cases):
    n = int(input())
    print(f"Fib({n}) = {fib[n]}")